### PR TITLE
feat: merge feature-inventory-collector branch and master(1.12.dev58)

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/alert-manager/alert-dashboard/modules/AlertStateWidget.vue [hunk 1]
```diff
@@ -1,3 +1,112 @@
...
+                    </div>
+                </div>
+                <!--list-->
+                <p-list-card :items="state.items"
+                             :loading="state.loading"
+                             :hoverable="true"
+                >
+                    <template #header>
+                        <div class="mobile-header">
+                            <p-checkbox v-model="state.isAssignedToMe">
```
Comment (piggggggggy) [inline apps/web/src/services/alert-manager/alert-dashboard/modules/AlertStateWidget.vue:76]: nitpick: `v-model` should be used with `argument`!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectDataButtonGroup.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<template>
+    <div class="collect-data-button-group">
+        <p-button style-type="tertiary"
+                  size="md"
+                  icon-left="ic_collect"
+                  class="collect-data-button"
+                  :class="{dependent: state.showStatus}"
+                  @click="handleClickCollectDataButton"
+        >
+            {{ $t('INVENTORY.COLLECTOR.DETAIL.COLLECT_DATA') }}
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectDataButtonGroup.vue:51]: nitpick: `$t` should be migrated to `const { t } = useI18n()`

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorPluginInfo.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            <template v-if="!props.showMinimal"
+                      #contents
+            >
+                <div class="collector-contents-wrapper">
+                    <div class="collector-contents">
+                        <p-field-title size="sm"
+                                       color="gray"
+                                       font-weight="regular"
+                        >
+                            {{ $t('INVENTORY.COLLECTOR.DETAIL.AUTO_UPGRADE') }}
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorPluginInfo.vue:35]: nitpick: same here! (`$t` to `useI18n`)

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorTags.vue [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                       :key="`${key}-${value}`"
+                       :key-item="{name: key}"
+                       :value-item="{name: value}"
+                       :deletable="false"
+                       style-type="primary"
+                       size="sm"
+                />
+            </template>
+            <p-empty v-else>
+                {{ $t('INVENTORY.COLLECTOR.DETAIL.NO_TAG') }}
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorTags.vue:43]: nitpick: same here (`$t` to `useI18n`)

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/PluginSummaryCards.vue [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+</script>
+
+<template>
+    <div class="plugin-summary-cards">
+        <div class="plugin-summary-card">
+            <p-field-title size="sm"
+                           color="gray"
+                           font-weight="regular"
+            >
+                {{ $t('INVENTORY.COLLECTOR.DETAIL.PLUGIN_ID') }}
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/PluginSummaryCards.vue:32]: nitpick: same here ($t to useI18n)

**Review by piggggggggy — APPROVED** [review]
I left some comments! Thanks~~~ LGTM!

**Comment by WANZARGEN** [comment]
I'll work your comments on the next PR!

**Pull request merged**